.jenkinsci-new/helpers/remove-function-lines.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# https://github.com/JeremyAgost/lcov-llvm-function-mishit-filter
+# https://github.com/linux-test-project/lcov/issues/30
+
+import argparse
+import subprocess
+
+def demangle(symbol):
+    p = subprocess.Popen(['c++filt','-n'], stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+    return p.communicate(input=symbol.encode())[0].decode().strip()
+
+def filter_lcov(lines, verbose=False):
+    defs, srcfile = {}, ''
+    for line in lines:
+        if line.startswith('SF:'):
+            defs = {}
+            srcfile = line[3:].strip()
+        elif line.startswith('end_of_record'):
+            defs = {}
+        elif line.startswith('FN:'):
+            lineno, symbol = line[3:].split(',')
+            if verbose:
+                defs[lineno] = demangle(symbol)
+            else:
+                defs[lineno] = True
+        elif line.startswith('DA:'):
+            lineno = line[3:].split(',')[0]
+            if lineno in defs:
+                if verbose:
+                    printf('Ignoring: {srcfile}:{lineno}:{defs[lineno]}')
```

Comment:
does this work? I see such syntax for the first time and did not find any docs on such `printf` function.

Reply:
this did not work on my setup. I suggest removing verbose stuff: srcfile, demangle, printf, cli arg, etc. because we do not use it and it is bogous.